yarn.lock

```diff
@@ -1070,62 +1070,63 @@
     human-id "^1.0.2"
     prettier "^1.18.2"
 
-"@d4kmor/cli@^0.2.5":
-  version "0.2.5"
-  resolved "https://registry.yarnpkg.com/@d4kmor/cli/-/cli-0.2.5.tgz#04d2d6ed0b2baf47ed6d69502f8de172d8e73d83"
-  integrity sha512-uoh5xF5GyYXIwM7LyuvRQQdqas2bTFpEr6fmS9uSaJU7DutiXCxumwSd0uB4cskWGjhdXvpfBvXF2qSBpmeNwQ==
+"@d4kmor/cli@^0.3.2":
+  version "0.3.2"
+  resolved "https://registry.npmjs.org/@d4kmor/cli/-/cli-0.3.2.tgz#5736b6824655bad06a0b5eda214af4d12b1cd400"
```

Comment:
why did the registry change?

Reply:
huh... no clue 😅 
maybe I had connection issues and it did fall back to npm registry instead of yarn? 🤷‍♂️

Reply:
fixed